public/content/en/basics/functions.md

```diff
@@ -1,17 +1,15 @@
 # Functions
 
-You've seen one function already: `main()` - the starting point of every
-D program. A function may return something - or be declared with
-`void` if nothing is returned - and an arbitrary number of parameters.
+One function has already been introduced: `main()` - the starting point of every
+D program. A function may return a value - or be declared with
+`void` if nothing is returned - and accept an arbitrary number of input parameters.
 
     int add(int lhs, int rhs) {
         return lhs + rhs;
     }
 
-If the return type is defined as `auto` the D compiler infers the return
-type automatically. If the types of different `return` statements within
-the function's body don't match the compiler will certainly make you
-aware of that.
+If the return type is defined as `auto`, the D compiler infers the return
+type automatically. Hence multiple `return` statements must return the same type.
```

Comment:
... `return` statements must return **values with compatible types**. See https://dpaste.dzfl.pl/76d3268d5a66 for reference.